scripts/wait-for-npm-release.sh

```diff
@@ -19,7 +19,7 @@ fi
 expected_version=$(node -p "require('./package.json').version")
 while [ true ]
 do
-  released_version=$(npm show hypothesis dist-tags.$dist_tag)
+  released_version=$(yarn info -s hypothesis dist-tags.$dist_tag)
```

Comment:
This change looks fine. `yarn info -s hypothesis dist-tags.latest` produces the same result as `npm show`.

Reply:
I confirm it does produce the same result.